docs/index.md

```diff
@@ -0,0 +1,134 @@
+---
+organization: Turbot
+category: ["software development"]
```

Comment:
Is this the correct category?

Reply:
Aiven is a fully managed, open-source cloud data platform. Should we use `managed cloud`?

Reply:
We have to use a category the Hub supports today (can be seen on left hand menu in https://hub.steampipe.io/plugins Maybe Public Cloud or SaaS?

Reply:
I think we should go with SaaS.